ui/app/src/Views/Settings/Views.elm

```diff
@@ -0,0 +1,37 @@
+module Views.Settings.Views exposing (view)
+
+import Html exposing (..)
+import Html.Attributes exposing (class, for, id, selected, value)
+import Html.Events exposing (..)
+import Views.Settings.Types exposing (Model, SettingsMsg(..))
+
+
+view : Model -> Html SettingsMsg
+view model =
+    div []
+        [ h1 []
+            [ text "Settings" ]
+        , div [ class "form-group" ]
+            [ label
+                [ for "select" ]
+                [ text "Start of Week:" ]
+            , select
+                [ onInput UpdateStartWeekAtMonday, id "select", class "form-control" ]
+                [ option
+                    [ value "1"
+                    , selected
+                        (model.startOfWeek == 1)
+                    ]
+                    [ text "Monday" ]
+                , option
+                    [ value "7"
+                    , selected
+                        (model.startOfWeek == 7)
+                    ]
+                    [ text "Sunday" ]
+                ]
+            , small [ class "form-text text-muted" ]
+                [ text "Note: This setting is saved in local storage of your browser"
+                ]
+            ]
+        ]
```

Comment:
Can you share the screenshot once you fix the spacing issue reported by @simonpasquier ? I would probably check the layout of the silence form for an inspiration.

Reply:
<img width="1163" alt="Current Layout of Settings Page" src="https://user-images.githubusercontent.com/2174923/195992579-f641e88c-3e9b-482d-8385-b5d87e76e140.png">

Screenshot of current layout of settings page

Reply:
I think it would look much nicer as two radio buttons:

🔘 Monday 🔘 Sunday

But if that's cumbersome, could you make the dropdown shorter?